docs/UserGuide.md

```diff
@@ -114,11 +114,12 @@ Format: `list`
 
 Adds a patient into HealthSync, with the given patient information.
 
-* A patient's name and IC are required when creating a new entry into HealthSync.
-* All the compulsory fields must be provided.
-* Optional fields like appointment and medical history need not be provided.
+* A patient's [name](#name) and [IC](#ID) are required when creating a new entry into HealthSync.
+* All the compulsory [fields](#fields) must be provided.
+* Optional fields like [Appointment](#appointment) and [medical history](#medical-history) need not be provided.
 
-Format: `add n/NAME id/ID_NUMBER [field] ...`
+[Format](#command-format):
+`add n/NAME id/ID_NUMBER p/PHONE_NUMBER e/EMAIL_ADDRESS a/ADDRESS [m/MEDICAL_HISTORY]... [ap/APPT]`
```

Comment:
Perhaps we should change `IC` to `ID` here to avoid confusion. Can also opt to provide an explanation like, "Patient's NRIC will be used as the ID_NUMBER".